<?xml version="1.0"?>
  <!-- camera version -->
<launch>
  <arg name="vocabulary"/>
  <arg name="setting" />
  <!-- Leg detection -->
  <node name="bundle_adjustment" pkg="orbslam" type="bundle_adjustment_node" 
        output="screen" />

  <node name="orb_extractor_node" pkg="orbslam" type="orb_extractor_node"
    args="$(arg vocabulary) $(arg setting)" />

  <node name="orb_slam_div2" pkg="orbslam" type="orbslam_node_div2"
    args="$(arg vocabulary) $(arg setting)" output="screen"  />

</launch>
